### DISCUSSION TITLE
The tabs have different style than the default style as in the docs

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [fuma-nama/fumadocs#556] [Q&A] [answered]
![image](https://github.com/fuma-nama/fumadocs/assets/42802922/891e9635-7ae5-4348-8875-bdfa05e4e512)

Here is the [code](https://github.com/Ali-Hussein-dev/indie-ui/blob/main/content/docs/eye-catching-buttons.mdx)

**Comment by fuma-nama** [comment, marked as answer]
You have to update to the latest version of Fumadocs, we add UI improvements frequently
